tests/Carbon/TestingAidsTest.php

```diff
@@ -269,4 +269,17 @@ public function testSetTestNow()
 
         $this->assertTrue($n2 > $n1);
     }
+
+    public function testWithTestNow()
+    {
+        $self = $this;
+        $test_now = '2020-09-16 10:20:00';
+        Carbon::withTestNow($test_now, static function () use ($self, $test_now) {
+            $current_time = Carbon::now();
+            $self->assertSame($test_now, $current_time->format('Y-m-d H:i:s'));
+        });
+
+        $current_time = Carbon::now();
+        $this->assertNotEquals($test_now, $current_time->format('Y-m-d H:i:s'));
```

Comment:
Actually I was expecting here to get the real now.

+ Please use camelCase for consistency: `$currentTime`, `$testNow`

Reply:
I'll update variable names.  That's an habit from our local style guide.

And this does get the actual now.  I am testing that it doesn't equal the test time any more.  This would only fail one second if we ran this in the past. ;-)

Reply:
👍